.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,37 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+blank_issues_enabled: true
+contact_links:
```

Comment:
Very interested to hear your thoughts on what order of contact links makes the most sense.

Reply:
Maybe "Ask" and "Report" links should be moved up and "Learn Swift" should be last?

Reply:
> Maybe "Ask" and "Report" links should be moved up

To the top, before "Discuss an idea" and "Formally propose a change"?